jjb/device/device-sdk-c.yaml

```diff
@@ -3,16 +3,15 @@
     name: device-sdk-c
     project-name: device-sdk-c
     project: device-sdk-c
+    build-node: ubuntu18.04-docker-arm64-4c-2g
     mvn-settings: device-sdk-c-settings
     archive-artifacts: ''
-    pre_build_script: 'make prepare && make test'
-    build_script: 'make stage && make verify'
+    pre_build_script: ''
+    build_script: './scripts/build.sh'
     cron: 'H 11 * * *'
     stream:
       - 'master':
-          branch: 'master'
-      - 'edinburgh':
-          branch: 'edinburgh'            
+          branch: 'master'          
     jobs:
-      - '{project-name}-cmake-stage-{stream}'
-      - '{project-name}-cmake-verify-{stream}'
\ No newline at end of file
+      - '{project-name}-cmake-verify-{stream}'

```

Comment:
Because this job name string is shared for two templates (one for Github and one for Gerrit) we will need to use the id.
So github-cmake-verify and github-cmake-stage

Though, I'm not sure we will need stage..since this is an sdk.  Will there be artifacts staged?

Reply:
the only artifact is the following:  'device-sdk-c/build/release/*.gz'

Reply:
@JPWKU why do we reference github-cmake ?  Is that just an example project name you're referring to?

Reply:
I think I I figured out what you were referring to - tried it with {project-name} and without - using the id (github-cmake-verify & github-cmake-stage).  No luck.